wordpress-mobile/WordPress-Android
# Remove (Yet! We're working on it) from the prompt

Fixes #11000 

### Test

1.	Fresh install the WordPress app.
2.	Login and go to **Reader** tab.
3.	Tap filter dropdown list in toolbar.
4.	Tap Discover item in list.
5.	Touch **Add to saved posts** action (bookmark icon) in the saved publication. 
6.	Notice the alert **Save Posts for Later**. It should not have the text `(Yet! We're working on it) `

P.S. This alert will appear only once when user creates a first bookmark.


| Before | After |
| ------------- | ------------- |
|![Screenshot_20191223-074500](https://user-images.githubusercontent.com/1405144/71331793-af349f80-2559-11ea-8f87-5573034bc2c1.png)| ![Screenshot_20191223-075153](https://user-images.githubusercontent.com/1405144/71331859-e6a34c00-2559-11ea-8698-28ff6042528d.png)|

### Review

Only one developer is required to review these changes, but anyone can perform the review.
